Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Action/EditorActionContext.h

```diff
@@ -23,13 +30,27 @@ namespace AzToolsFramework
         EditorActionContext() = default;
         EditorActionContext(AZStd::string identifier, AZStd::string name, AZStd::string parentIdentifier, QWidget* widget);
 
+        bool HasMode(const AZStd::string& modeIdentifier);
+        void AddMode(AZStd::string modeIdentifier);
+
+        AZStd::string GetActiveMode();
+        bool SetActiveMode(AZStd::string modeIdentifier);
+
+        void AddAction(AZStd::string actionIdentifier);
+        void GetActionIdentifiers(const AZStd::function<bool(const AZStd::string&)>& callback) const;
```

Comment:
Nitpick: Maybe a different name for this function since it's not actually returning anything. Maybe `IterateActionIdentifiers` or `InvokeOnActionIdentifiers` ?

Reply:
I had the same doubt but I was mimicking `GetEntityIds` from the Instance class that works the exact same way.
I like `IterateActionIdentifiers` though.